### DISCUSSION TITLE
Hide Search Bar once the value is been selected from dropdown

### DISCUSSION THREAD

**Discussion by aishwarya2392** [dowjones/react-dropdown-tree-select#467] [General]
Im trying to use this a single select tree dropdown, so whenever I select a value from the dropdown, I need the Search bar to be hidden.

Or is it possible to show the dropdown inside the Search bar similar to react select?
Now i will have to manually hide using css, I havent tried if its possible or not.

**Comment by mrchief** [comment]
> Now i will have to manually hide using css,

Well, that is by design. You have HTML, CSS and JS at your disposal - why use JS all the time? :)

**Reply by aishwarya2392** [reply]
Oh yeah, I have been doing that only now :) but just wanted to ask if there is any possibility of having it in the package itself, if this is been used as a single select tree dropdown.
Anyway thanks a lot.

**Reply by mrchief** [reply]
Doing everything via JS bloats the bundle size and reduces agility. Allowing flexibility for one of use cases is integral part of this component and the CSS has been carefully designed to make these type of changes easily. So it is part of the package :)

**Reply by aishwarya2392** [reply]
Got it and it wasnt so difficult to hide the search bar via css, so i think im good :)
